### ISSUE TITLE
Variable name checker should be an init option

### ISSUE THREAD

**Issue by mark-friedman** [mit-cml/blockly-plugins#12]
`Blockly.LexicalVariable.checkIdentifier` should become an input to the plugin since right now it’s set to match Kawa’s identifier rules.  

Note that Blockly’s default appears to be to allow anything as a variable name!  I think they might then fix things up in the code generator.

**Comment by Nishantware** [comment]
can you give a screenshots to more clarity ?

**Comment by DuyLe0710edu** [comment]
Hi Mark, I am Duy and a sophomore and I will be interested in solving this issues, I think the possible solution could be to make the configurable check Identifier and we can possibly parse the validation function.

**Comment by mark-friedman** [comment]
@DuyLe0710edu, can you give some more details on your proposed solution?

**Comment by DuyLe0710edu** [comment]
@mark-friedman yes, of course, with my understanding , the problem is the variables are hardcoded using the Kawa's identifier rule and the rule are fixed. My solution is to create a plugin initialization that can parse the custom validator as an option ( assign options = {}, Lexical.checkIdentifier = options.checkIdentifier ) then we can use that plugin instead and we can modify the rule.

**Comment by DuyLe0710edu** [comment]
Can you assign me this problem so I can try to apply my solution ??

**Comment by mark-friedman** [comment]
> Can you assign me this problem so I can try to apply my solution ??

Done.  I look forward to seeing your pull request.

**Comment by DuyLe0710edu** [comment]
Thanks, I will start working on this
